gcloud_requests/proxy.py

```
@@ -34,10 +34,17 @@ class RequestsProxy(object):
 
     #: Determines how retries should be handled by this proxy.
-    RETRY_CONFIG = Retry(
-        total=10, connect=10, read=5,
-        method_whitelist=Retry.DEFAULT_METHOD_WHITELIST | frozenset(["POST"])
-    )
+    # Handle compatibility between old urllib3 (method_whitelist) and new (allowed_methods)
+    _retry_kwargs = {
+        "total": 10,
+        "connect": 10,
+        "read": 5,
+    }
```

Ruff (0.14.7)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)